modules/apps/product-navigation/product-navigation-product-menu-impl/src/main/java/com/liferay/product/navigation/product/menu/internal/helper/ProductNavigationProductMenuHelperImpl.java

```diff
@@ -43,7 +43,7 @@ public boolean isShowProductMenu(HttpServletRequest httpServletRequest) {
 			(ThemeDisplay)httpServletRequest.getAttribute(
 				WebKeys.THEME_DISPLAY);
 
-		if (!themeDisplay.isSignedIn() || !themeDisplay.isShowControlMenu()) {
+		if (!themeDisplay.isSignedIn()) {
```

Comment:
Could this result in isShowProductMenu being true when isShowControlMenu is false?